### DISCUSSION TITLE
I failed when I used Parameter Identification with the following result ：Simulation did not produce results

### DISCUSSION THREAD

**Discussion by doctorcsu** [Open-Systems-Pharmacology/Forum#1772] [Questions & Problems]
![屏幕截图 2024-06-04 202102](https://github.com/Open-Systems-Pharmacology/Forum/assets/128056597/da69bac1-8d9b-4aed-8958-c12a93f424f4)

**Comment by SteTindal** [comment]
I have had similar issues when parameter value range minimums were set to 0 or when there were more than 8 parameters.

**Comment by msevestre** [comment]
@doctorcsu When you go over with the mouse on Faulted or the Message, do you see more info?

The fact that Infinity is shown and number of evaluation is 0 means that the simulation does not even seem to run with the start value provided. Can you actually run your simulation? Is this PK-Sim or MoBi? 

@SteTindal This should not be the case. The number of parameters should not play a role unless the PI is setting some parameters to values that make the simulation diverge.

**Comment by bruni-cris** [comment]
I had a similar error when I tried to input several simulations (n=16) together in PK-Sim.

![image](https://github.com/user-attachments/assets/e1f2b8f6-e2fa-4ef7-85ba-351c94829f91)

@msevestre do you have any insights about what could be the problem?

When I did the PI for the drug residuals in tissues (n=8) it worked pretty well, but when I combined tissue and plasma simulations to perform a second PI (n=16 simulations), this error message appeared.

**Reply by msevestre** [reply]
Would you be willing to share the project? 
If yes , this would be extremely useful for us to understand. I can also send you a link for private upload if you prefer.

**Reply by bruni-cris** [reply]
Yes, please send me the private link, so I can share more information

**Reply by msevestre** [reply]
https://aedginton.egnyte.com/ul/HXTLDEQtWu

**Reply by bruni-cris** [reply]
Just sent it. Thank you very much!
